### ISSUE TITLE
WDM should work without internet

### ISSUE THREAD

**Issue by asolntsev** [bonigarcia/webdrivermanager#415]
... if I already have the latest chromedriver binary in local cache.
**Description of the problem**: 
1. I already have latests chromedriver in cache
2. I have no internet connection for several hours (plane, airport, outage etc.) I want to use this time effectively and run my UI tests. :)
3. I cannot start tests because WDM still tries to connect to internet on every run. :(

**Browser and version**: doesn't matter
**Operating system**: doesn't matter
**WebDriverManager version**: 3.7.1
**WebDriverManager use**: 

```java
WebDriverManager webDriverManager = WebDriverManager.chromedriver().avoidPreferences();
webDriverManager.setup();
```

**Error trace**: 

```
"main" #1 prio=5 os_prio=0 cpu=2311,38ms elapsed=122,53s tid=0x00007f39e8019000 nid=0x29ab runnable  [0x00007f39ef673000]
   java.lang.Thread.State: RUNNABLE
at java.net.Inet6AddressImpl.lookupAllHostAddr(java.base@11.0.5-ea/Native Method)
at java.net.InetAddress$PlatformNameService.lookupAllHostAddr(java.base@11.0.5-ea/InetAddress.java:929)
at java.net.InetAddress.getAddressesFromNameService(java.base@11.0.5-ea/InetAddress.java:1515)
at java.net.InetAddress$NameServiceAddresses.get(java.base@11.0.5-ea/InetAddress.java:848)
- locked <0x000000071744f8b0> (a java.net.InetAddress$NameServiceAddresses)
at java.net.InetAddress.getAllByName0(java.base@11.0.5-ea/InetAddress.java:1505)
at java.net.InetAddress.getAllByName(java.base@11.0.5-ea/InetAddress.java:1364)
at java.net.InetAddress.getAllByName(java.base@11.0.5-ea/InetAddress.java:1298)
at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:374)
at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
at io.github.bonigarcia.wdm.HttpClient.execute(HttpClient.java:169)
at io.github.bonigarcia.wdm.ChromeDriverManager.getLatestVersion(ChromeDriverManager.java:119)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:515)
at io.github.bonigarcia.wdm.WebDriverManager.handleException(WebDriverManager.java:751)
at io.github.bonigarcia.wdm.WebDriverManager.manage(WebDriverManager.java:563)
at io.github.bonigarcia.wdm.WebDriverManager.setup(WebDriverManager.java:260)
- locked <0x0000000719b5b8d8> (a io.github.bonigarcia.wdm.ChromeDriverManager)
at my.com.tests.ui.ChromeFactory.createDriver(ChromeFactory.java:46)
```

**Comment by bonigarcia** [comment]
I agree. Let me check for the next version.

**Comment by bonigarcia** [comment]
Included in version 3.8.0.
